ci-operator/step-registry/upi/libvirt/heterogeneous/pre/upi-libvirt-heterogeneous-pre-chain.yaml

```diff
@@ -1,12 +1,27 @@
 chain:
   as: upi-libvirt-heterogeneous-pre
   steps:
-  - ref: upi-libvirt-conf
+  - ref: upi-libvirt-cleanup-pre
+  - chain: upi-conf-libvirt
```

Comment:
This is Dominik's chain. Are we removing `upi-libvirt-conf` (Dominik's old conf steps) in favor of our updated `upi-conf-libvirt` flow? If not we probably want to separate out these changes into their own PR since they're affecting Dominik's workflow, not ours.